db.py

```diff
@@ -43,6 +43,7 @@ def start():
         c.execute("ALTER TABLE image_hashes ADD hash_color TEXT NOT NULL")
         c.execute("ALTER TABLE meme_stats ADD meme_rating INTEGER")
         c.execute("ALTER TABLE meme_stats ADD rating_count INTEGER")
+        c.execute("ALTER TABLE meme_stats ADD meme_reasoning TEXT")
```

Comment:
Dirty secret, this won't actually work. The first ALTER TABLE generates an exception and the rest get ignored. So each line needs its own try except, or something similar.